libs/model/src/services/tokenBalanceCache/providers/get_eth_balances.ts

```diff
@@ -1,5 +1,6 @@
 import { logger } from '@hicommonwealth/logging';
-import { toBN } from 'web3-utils';
+import { ethers } from 'ethers';
```

Comment:
Per https://github.com/hicommonwealth/commonwealth/pull/7416#issuecomment-2067305714, we should not add `ethers`. Web3-utils has been updated to the latest version so we should update accordingly.

Reply:
Looks like 7285-bump-web3js-to-v4xx has been merged so we can excise imports of ethers here.

Reply:
Will be resolved after `libs` is upgraded from web3.js v1 to v4, at which point we can revert the couple of places that use ethers.